StackOverflowError exception in the AnimationController when actions are used

Hi,
Thank you for developing and supporting this great framework.

I've faced some interesting issue and was able to make it stable reproducible.
Please see details below:

#### Reproduction steps/code

```java
        AssetManager assets = new AssetManager();
        assets.load("3d-models/animation-transition-in-details.g3db", Model.class);
        assets.finishLoading();
        Model model = assets.get("3d-models/animation-transition-in-details.g3db");
        ModelInstance modelInstance = new ModelInstance(model);
        AnimationController animationController = new AnimationController(modelInstance);
        animationController.setAnimation("Suzanne|init", -1);
        animationController.update(animationController.current.duration);
        animationController.action("Suzanne|first", 1, 1f, null, 1f);
        animationController.update(animationController.current.duration);
        animationController.update(0.001f);
        // This invocation throws
        animationController.action("Suzanne|second", 1, 1f, null, 1f);
```

Related model file: [animation-transition-in-details.zip](https://github.com/libgdx/libgdx/files/5111920/animation-transition-in-details.zip)


#### Version of LibGDX and/or relevant dependencies
1.9.11

#### Stacktrace
```java
Exception in thread "LWJGL Application" com.badlogic.gdx.utils.GdxRuntimeException: java.lang.StackOverflowError
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:134)
Caused by: java.lang.StackOverflowError
	at com.badlogic.gdx.graphics.g3d.utils.AnimationController.queue(AnimationController.java:425)
	at com.badlogic.gdx.graphics.g3d.utils.AnimationController.animate(AnimationController.java:363)
	at com.badlogic.gdx.graphics.g3d.utils.AnimationController.queue(AnimationController.java:425)
	at com.badlogic.gdx.graphics.g3d.utils.AnimationController.animate(AnimationController.java:363)
```

#### Please select the affected platforms
- [ ] Android
- [ ] iOS (robovm)
- [ ] iOS (MOE)
- [ ] HTML/GWT
- [X] Windows
- [ ] Linux
- [ ] MacOS
_Have not tested on other platforms but think it can be reproducible as well_

--- a/gdx/src/com/badlogic/gdx/graphics/g3d/utils/AnimationController.java
+++ b/gdx/src/com/badlogic/gdx/graphics/g3d/utils/AnimationController.java
@@ -357,7 +357,7 @@ public class AnimationController extends BaseAnimationController {
 
 	/** Changes the current animation by blending the new on top of the old during the transition time. */
 	protected AnimationDesc animate (final AnimationDesc anim, float transitionTime) {
-		if (current == null)
+		if (current == null || current.loopCount == 0)
 			current = anim;
 		else if (inAction)
 			queue(anim, transitionTime);

